docs/_config.yml

```diff
@@ -3,7 +3,7 @@
 
 title: Ard
 author: National Renewable Energy Laboratory
```

Comment:
I think actual author names are usually specified, with other attribution going to NREL, not sure though